docs/radio.rst

```diff
@@ -132,6 +133,27 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_all()
+
+    Returns a tuple containing four values representing the next incoming
+    message on the message queue. If there no pending messages then ``None``
+    is returned.
+
+    The four values in the tuple represent:
+
+    * the next incoming message on the message queue as bytes.
+    * the RSSI signal strength: a value between 0 (strongest) and 255 (weakest).
```

Comment:
Not sure I understand. I'm concerned that we can say 0-255 measured as [insert measurement scale here].